src/popup/popup.js

```diff
@@ -75,7 +73,6 @@ async function initApp() {
   Vue.use(VModal, { dialog: true });
   Vue.use(VueRouter);
   Vue.use(ToggleButton);
-  Vue.use(TextareaAutosize);
```

Comment:
Do you know where do we use this one?

Reply:
We don't use it anywhere!

If we look for it on `master`:

```
λ rg TextareaAutosize
src/popup/popup.js
13:import TextareaAutosize from "vue-textarea-autosize";
78:  Vue.use(TextareaAutosize);
```

Reply:
I tested and removing TextareaAutosize breaks the alias note editing. In the current version, the note automatically resizes when user enters
<img width="435" alt="Screen Shot 2022-06-03 at 15 20 53" src="https://user-images.githubusercontent.com/3167780/171862731-1c45aba2-8ab2-4475-8d25-a6f2d5e463bb.png">

whereas in this version, the note doesn't even allow editing ... (it's hard to see from a screenshot)

<img width="438" alt="image" src="https://user-images.githubusercontent.com/3167780/171862819-f0672c2f-bcd6-4bb4-982b-b8f16cfe6844.png">

Reply:
Fixed it!